test/functional/p2p_segwit.py

```diff
@@ -224,7 +222,6 @@ def skip_test_if_missing_module(self):
     def setup_network(self):
```

Comment:
This method override can be removed entirely, since it's doing the same as the default `setup_network()` in test_framework.py

Reply:
Done